src/bin/pgcopydb/ld_transform.c

```diff
@@ -2227,12 +2227,15 @@ stream_write_update(FILE *out, LogicalMessageUpdate *update)
 						  update->table.relname);
 		int pos = 0;
 
+		int skippedColumns = 0;
```

Comment:
Can re rename to `skipColumnCount` or something? at the moment this reads more like a boolean value to me.